lib/commands/serve.js

```diff
@@ -21,9 +18,9 @@ module.exports = Command.extend({
     {
       name: 'port',
       type: Number,
-      default: defaultPort,
+      default: process.env.PORT,
```

Comment:
```suggestion
      default: process.env.PORT || DEFAULT_PORT,
```

`$PORT` may not exist, so we should fall back to the true default